client/src/templates/Challenges/rechallenge/builders.js

```diff
@@ -89,10 +91,7 @@ A required file can not have both a src and a link: src = ${src}, link = ${link}
 
   const source = files.reduce((source, file) => {
     if (!indexHtml) return source.concat(file.contents, htmlCatch);
-    if (
-      indexHtml.includes.includes(file.history[0]) ||
-      file.history[0] === 'index.html'
-    ) {
+    if (indexHtml.includes.includes(file.history[0]) || wasHtmlFile(file)) {
```

Comment:
Any chance we can change this to not _include_ a property name the same as an in-built function?

Reply:
Sure. How about importedFiles?

Reply:
Is it not just a duplicate of `files[<file>].path`? If "no", then that should work.

Reply:
Nope, it's an array containing `index.css` and/or `index.js`, which the transformer needs in order to include or exclude those files from the preview and tests.